map_it_test.go

```diff
@@ -870,6 +969,36 @@ func TestMap_EntryNotifiedEventToKeyAndPredicate(t *testing.T) {
 	})
 }
 
+func TestMap_EntryNotifiedEventToKeyAndPredicateWithSpecificHandler(t *testing.T) {
+	cbCallback := func(config *hz.Config) {
+		config.Serialization.SetPortableFactories(it.SamplePortableFactory{})
+	}
+	it.MapTesterWithConfig(t, cbCallback, func(t *testing.T, m *hz.Map) {
+		const totalCallCount = int32(1)
+		callCount := int32(0)
+		handler := func(event *hz.EntryNotified) {
+			atomic.AddInt32(&callCount, 1)
+		}
+		listenerConfig := hz.MapEntryListenerConfig{
+			IncludeValue: true,
+			Key:          "k1",
+			Predicate:    predicate.Equal("A", "foo"),
+		}
+		listenerConfig.SetEntryAddedListener(handler)
+		if _, err := m.AddEntryListener(context.Background(), listenerConfig); err != nil {
+			t.Fatal(err)
+		}
+		it.MustValue(m.Put(context.Background(), "k1", &it.SamplePortable{A: "foo", B: 10}))
+		it.MustValue(m.Put(context.Background(), "k1", &it.SamplePortable{A: "bar", B: 10}))
```

Comment:
I think that will trigger `Updated` not `Added` event right? That is why `callCount` is 1.

Reply:
Yes exactly